public/js/pimcore/element/helpers/gridCellEditor.js

```diff
@@ -121,7 +117,11 @@ Ext.define('pimcore.element.helpers.gridCellEditor', {
                     text: t("save"),
                     iconCls: 'pimcore_icon_save',
                     handler: function() {
-                        var newValue = tag.getCellEditValue();
+                        if (typeof tag.isDirty === 'function' && tag.isDirty() === false) {
```

Comment:
@kingjia90 It's the same as in the admin-ui https://github.com/pimcore/admin-ui-classic-bundle/blob/1976b8f869ad450e1b3d2b9e4aebcc47f3487e7e/public/js/pimcore/asset/metadata/tags/manyToOneRelation.js#L198 I thought it would be good to have the same functionality in every tag where it's needed